Exported service doesn't exist

https://github.com/offirgolan/ember-parachute/blob/master/app/services/qp.js#L1

The file cannot be found when injecting the service, i.e.

```
ember-metal.js:3988 Error: Could not find module `ember-parachute/services/qp` imported from `bwt/services/qp`
    at missingModule (loader.js:247)
    at findModule (loader.js:258)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at requireModule (loader.js:24)
    at Class._extractDefaultExport (index.js:392)
    at Class.resolveOther (index.js:113)
    at Class.superWrapper [as resolveOther] (ember-utils.js:423)
    at Class.resolve (resolver.js:164)
    at resolve (container.js:873)
```

I'd submit a fix if I knew where the service was..

--- a/app/services/qp.js
+++ b/app/services/qp.js
@@ -1,1 +1,0 @@
-export { default } from 'ember-parachute/services/qp';

